environment.yml

```diff
@@ -1,7 +1,7 @@
 channels:
 - conda-forge
 dependencies:
-- python >=3.10
+- python =3.9
```

Comment:
great! we should add specific versions for all dependencies as we make improvements (e.g. I know some older versions of numpy don't work)